apps/qa/src/checkbook/checkbook.py

```diff
@@ -32,7 +33,8 @@ def checkbook():
         ),
     )
 
-    data = get_data()
+    draft, selection = sidebar.data_selection("db-checkbook")
```

Comment:
If we go with this sort of model - "output type" (draft v publish) and "output label" ("21v1.2", "fvk-draft-and-publish-in-s3"), we should decide on terminology and I need to make consistent everywhere. And maybe this becomes a named tuple or something? As you'll see there's a lot of passing these around together

Reply:
just thoughts on terminology:
- multiple draft outputs could have the same version, but published outputs can't
- the same branch could build multiple versions. but that doesn't mean we plan on doing QA of that situation anytime soon, just saying it's possible to do
- in draft, I imagine a folder will always be named after the branch that build it? we don't seem to make branches like `21v1.2`
- **maybe these terms could work:**
  - `output_type`: the "environment" that an output is in (`draft` or `publish`)
  - `build_name`: the branch that built a draft output
  - `version`: the release version of an output

so when publishing a QAd build, an S3 folder like `draft/dm-build-pluto-today/output/` would be copied to `publish/23v4/output/`. both would have some form of a `version.txt` indicating that they're `23v4` builds

Reply:
from chatting, general conclusion was to use the term "key" (aligns with S3 terminology) to refer to the unique path to any build's output